lib/ansible/utils/module_docs_fragments/nios.py

```diff
@@ -70,7 +70,7 @@ class ModuleDocFragment(object):
           - Specifies the version of WAPI to use
           - Value can also be specified using C(INFOBLOX_WAP_VERSION) environment
             variable.
-        default: 1.4
+        default: 2.1
```

Comment:
you should note in the description that the default changed in version 2.8

Reply:
@bcoca I didn't get the exact query you are pointing me too, I have updated the default wapi version as we use wapi `2.1` from code end: https://github.com/ansible/ansible/blob/991a9dfa120b45d566e6f7aef428fe36f0372137/lib/ansible/module_utils/net_tools/nios/api.py#L69

Reply:
@bcoca I have updated the description as suggested, can you please review and approve.